locksmith/migrations/20190205161953-create_transactions.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Transactions', {
+      transactionHash: {
+        type: Sequelize.STRING,
+        primaryKey: true
+      },
+      createdAt: {
+        allowNull: false,
```

Comment:
do we set a default value here?

Reply:
This is provided by the framework, it will be the time of creation

Reply:
Ok, great. Wasn't sure. Maybe add a comment about that?